src/components/OptionsList/BaseOptionsList.js

```diff
@@ -167,6 +167,7 @@ class BaseOptionsList extends Component {
                 boldStyle={this.props.boldStyle}
                 isDisabled={this.props.isDisabled || section.isDisabled}
                 shouldHaveOptionSeparator={index > 0 && this.props.shouldHaveOptionSeparator}
+                disableInnerPadding={this.props.disableRowInnerPadding}
```

Comment:
I think it would be more clear if the prop names were the same?

Reply:
What do you think about this @cristipaval ?

Reply:
fixed.